cypress/support/pages/onboarding/newsletters_page.ts

```diff
@@ -4,7 +4,7 @@ class NewslettersPage extends OnboardingPage {
   static CONTENT = {
     ...OnboardingPage.CONTENT,
     NEWSLETTERS: {
-      MORNING_BRIEFING_UK: 'Guardian Morning Briefing',
```

Comment:
I'm not sure cypress mocks getting out of sync is that big of a deal?

As for the colors, I think this would already be broken on prod if `nameId` had changed. Looks like it hasn't:

![Screenshot 2022-05-19 at 10 45 45](https://user-images.githubusercontent.com/3338808/169264975-fba8a82f-bf4a-44d4-b074-4966a433f2a6.png)

Maybe we can rely on them to be static? e.g. "Down to Earth" still has the id `green-light`.